Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -869,22 +872,44 @@ protected virtual void CreateCollider()
                 bodyRigidbody.freezeRotation = true;
             }
 
-            if (bodyColliderContainer == null)
+            if (customBodyCollider == null)
             {
                 generateCollider = true;
-                bodyColliderContainer = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
+                customBodyCollider = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
 
-                bodyCollider = bodyColliderContainer.AddComponent<CapsuleCollider>();
+                bodyCollider = customBodyCollider.AddComponent<CapsuleCollider>();
                 bodyCollider.radius = bodyRadius;
 
+                customBodyCollider.gameObject.layer = LayerMask.NameToLayer("Ignore Raycast");
+            }
+            else
+            {
+                customBodyCollider.transform.SetParent(playArea);
```

Comment:
I'm guessing this means that the object has to exist in the scene and you can't use a prefab for the injected game object?